commands/autonomus_driving.py

```diff
@@ -0,0 +1,18 @@
+from wpilib.command import CommandGroup, WaitCommand
+
+from commands.move_timed import MoveTimed
+from commands.spin_timed import SpinTimed
+
+class AutonomusDriving(CommandGroup):
+
+    def __init__(self):
+        super().__init__()
+        self.addSequential(MoveTimed(1), 2.0)
+        self.addSequential(SpinTimed(0.25), 2.0)
+        self.addSequential(MoveTimed(1), 2.0)
+        self.addSequential(SpinTimed(0.25), 2.0)
+        self.addSequential(MoveTimed(1), 2.0)
+        self.addSequential(SpinTimed(0.25), 2.0)
+        self.addSequential(MoveTimed(1), 2.0)
+        self.addSequential(SpinTimed(2), 2.0)
```

Comment:
This is good for now, just know we are going to reimplement this in the future using Motion Profiling. In the meantime, feel free to learn what Motion Profiling is and [check this out.](https://github.com/robotpy/examples/blob/master/physics-pathfinder/robot.py)